src/config/src/meta/folder.rs

```diff
@@ -13,21 +13,11 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-use serde::{Deserialize, Serialize};
-use utoipa::ToSchema;
-
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
```

Comment:
Intentionally removing all serialization/deserialization logic since this is specific to HTP API